run_benchmark.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash

set -euo pipefail
```

This ensures that the script fails on any failing command.